example/lib/main.dart

```diff
@@ -24,6 +25,7 @@ class MyApp extends StatelessWidget {
       routes: {
         '/': (context) => DonutCounterScreen(),
         '/permissions': (context) => PermissionsScreen(),
+        '/test': (context) => TestScreen(),
```

Comment:
- [ ] If you renamed it, probably both of these also?  (Or if you have a reason not to, fine, just check the box to get it out of your way.)